apiary.apib

```diff
@@ -231,6 +231,138 @@ Get the current authenticated user.
             + language: `nl` (string)
             + function: `Sales` (string)
 
+# Group Activities
+
+## Calendar events [/events]
+
+### events.list [GET /events.list]
+
+Get a list of calendar events.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `255a988d-d4b6-47eb-a5b8-746c24b5171a`,`e41cf7cd-05cd-4f28-b138-d6d585c30c64` (array[string], optional)
+            + user_id: `5309c8c9-e313-47b0-90ba-6850c3dc3e33` (string, required)
+            + activity_type_id: `edd94120-63e2-4b10-80ed-fdfcfaa0a515` (string, optional)
+            + ends_after: `2017-01-01T00:00:00+00:00` (string, optional) - Start of the period for which to return events
+            + starts_before: `2018-01-01T00:00:00+00:00` (string, optional) - End of the period for which to return events
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field: `starts_at` (enum[string], required)
+                    + Members
+                        + starts_at
+                + order: `asc` (enum[string], optional)
+                    + Members
+                        + asc
+            + Default
+                + (object)
+                    + field: starts_at
+                    + order: asc
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `6d64488c-75d8-4de9-b140-2555566fa27d` (string)
+                + creator (object)
+                    + type: `user` (string)
+                    + id: `e51d1a0e-452e-4d5d-a2a1-2c8f75dde560` (string)
+                + todo (object)
+                    + type: `todo` (string)
+                    + id: `eb399573-e1ae-4377-bbaf-34358eb19831` (string)
+                + activity_type (object)
+                    + type: `activityType` (string)
+                    + id: `28158af3-2ef3-4178-ac8b-2542b1481af0` (string)
+                + topic (object)
```

Comment:
remove for now?

Reply:
or rename to work_type

Reply:
:ok: by @regularjack on 29/11/2017 14:44:56